### PR TITLE
allow to add dimension values comma separated in a text box

### PR DESCRIPTION
**Pull request by khushbu-maurya**

### CODE REVIEW

**Comment by admin-vizcentric** [comment]
There were no errors, go have a cup of coffee...

**Comment by admin-vizcentric** [comment]
There were no errors, go have a cup of coffee...

File: src/main/webapp/app/entities/flair-bi/filter/filter-element-grpc.component.js [hunk 1]
```diff
@@ -95,15 +97,15 @@
 
             if (filterParameters[vm.dimension.name].indexOf(filter) !== -1) {
                 filterParameters[vm.dimension.name].splice(filterParameters[vm.dimension.name].indexOf(filter.name), 1);
-                removeTagFromSelectedList({text:filter});
+                removeTagFromSelectedList({ text: filter });
             }
             else {
                 filterParameters[vm.dimension.name].push(filter);
-                if(!vm.dimension.selected){
+                if (!vm.dimension.selected) {
```
Comment (johibkhan2) [inline src/main/webapp/app/entities/flair-bi/filter/filter-element-grpc.component.js:104]: there is no change in code..if u want to format the code...pls create separate PR

File: src/main/webapp/app/entities/flair-bi/filter/filter-element-grpc.component.js [hunk 2]
```diff
@@ -95,15 +97,15 @@
...
                     vm.dimension.selected = [];
-                    vm.dimension.selected.push({text:filter});
-                }else{
```
Comment (johibkhan2) [inline src/main/webapp/app/entities/flair-bi/filter/filter-element-grpc.component.js:105]: there is no change in code..if u want to format the code...pls create separate PR

File: src/main/webapp/app/entities/flair-bi/filter/filter-element-grpc.component.js [hunk 3]
```diff
@@ -95,15 +97,15 @@
...
-                    vm.dimension.selected.push({text:filter});
+                    vm.dimension.selected.push({ text: filter });
+                } else {
+                    vm.dimension.selected.push({ text: filter });
                 }
```
Comment (johibkhan2) [inline src/main/webapp/app/entities/flair-bi/filter/filter-element-grpc.component.js:109]: there is no change in code..if u want to format the code...pls create separate PR

File: src/main/webapp/app/entities/flair-bi/filter/filter-element-grpc.component.js [hunk 4]
```diff
@@ -113,7 +115,7 @@
             };
             filterParametersService.saveSelectedFilter(filterParameters);
             vm.isActive(filter);
-            if(isFavouriteFilter())
+            if (isFavouriteFilter())
```
Comment (johibkhan2) [inline src/main/webapp/app/entities/flair-bi/filter/filter-element-grpc.component.js:118]: there is no change in code..if u want to format the code...pls create separate PR

File: src/main/webapp/content/scss/partials/_filter-pane.scss [hunk 5]
```diff
@@ -7,6 +7,9 @@
     tags-input.filter-input {
       margin: 0.5vw;
     }
+    i.active{
+      color:$blue;
```
Comment (johibkhan2) [inline src/main/webapp/content/scss/partials/_filter-pane.scss:11]: can u give meaningful name to var $blue..name should relate to functionality

**Pull request merged**